packages/oui-list/_mixins.less

```diff
@@ -197,7 +197,7 @@
       border-bottom: @separator-size solid @separator-color;
 
       &:last-child {
-          border-bottom: 0 none;
+        border-bottom: 0 none;
```

Comment:
It's not the point here @NicoPennec but yes, `none` would be better. Maybe stylelint will be able to help us with that. Soon.